Public/Threads/Add-ThreadMessage.ps1

```diff
@@ -207,6 +210,16 @@ function Add-ThreadMessage {
         }
         #endregion
 
+        #region Wait for good time to send API request
+        if ($WaitForRunComplete) {
+            $runs = Get-ThreadRun -ThreadId $ThreadId -All
+            foreach ($run in $runs) {
+                Write-Host "Waiting for the run to complete. Run ID: $($run.id)"
+                Wait-ThreadRun -Run $run @CommonParams

```

Comment:
Since Wait-ThreadRun outputs a Thread object, the output must be suppressed here.
```suggestion
                $null = Wait-ThreadRun -Run $run @CommonParams
```